### ISSUE TITLE
1.3.0 causes Ember builds to fail

### ISSUE THREAD

**Issue by astronomersiva** [substack/node-resolve#119]
```
Error: Cannot find module '{module_name}' from '{project_path}' with extension in [".js"]
    at Function.module.exports [as sync] ({project_path}/resolve/lib/sync.js:38:15)
```

Related issue on Ember-CLI: https://github.com/ember-cli/ember-cli/issues/6807

**Comment by ljharb** [comment]
Thanks for the report - taking a look now.

**Comment by ljharb** [comment]
I've set "latest" to v1.2.0 while I try to figure out what broke.

**Comment by zsoltvad** [comment]
Same issue appeared for angular-cli.

```js
[1] Cannot find module '@types/cucumber' from '{project_path}' with extension in [".js"]
[1] Error: Cannot find module '@types/cucumber' from '{project_path}' with extension in [".js"]
[1]     at Function.module.exports [as sync] ({project_path}\node_modules\resolve\lib\sync.js:38:15)
[1]     at AddonDiscovery.resolvePackage ({project_path}\node_modules\angular-cli\lib\models\addon-discovery.js:136:35)
[1]     at AddonDiscovery.<anonymous> ({project_path}\node_modules\angular-cli\lib\models\addon-discovery.js:111:28)
[1]     at Array.map (native)
[1]     at AddonDiscovery.discoverFromDependencies ({project_path}\node_modules\angular-cli\lib\models\addon-discovery.js:109:68)
[1]     at AddonDiscovery.discoverProjectAddons ({project_path}\node_modules\angular-cli\lib\models\addon-discovery.js:48:29)
[1]     at Project.discoverAddons ({project_path}\node_modules\angular-cli\lib\models\project.js:353:40)
[1]     at Project.initializeAddons ({project_path}\node_modules\angular-cli\lib\models\project.js:372:8)
[1]     at Project.eachAddonCommand ({project_path}\node_modules\angular-cli\lib\models\project.js:425:10)
[1]     at module.exports ({project_path}\node_modules\angular-cli\lib\cli\lookup-command.js:33:13)
[1]     at CLI.<anonymous> ({project_path}\node_modules\angular-cli\lib\cli\cli.js:34:26)
[1]     at tryCatch ({project_path}\node_modules\rsvp\dist\rsvp.js:538:12)
[1]     at invokeCallback ({project_path}\node_modules\rsvp\dist\rsvp.js:553:13)
[1]     at publish ({project_path}\node_modules\rsvp\dist\rsvp.js:521:7)
[1]     at flush ({project_path}\node_modules\rsvp\dist\rsvp.js:2373:5)
[1]     at _combinedTickCallback (internal/process/next_tick.js:67:7)
[1]     at process._tickCallback (internal/process/next_tick.js:98:9)
[1]

`

**Comment by ljharb** [comment]
Reproduced locally; I've bisected it down to 1260d9d1e2f55efb514540db9aa1b3d679f9db10.

**Comment by ljharb** [comment]
OK, so in ember-cli this is because they were literally checking the error message [here](https://github.com/ember-cli/ember-cli/blob/master/lib/models/addon-discovery.js#L149).

I assume the same problem is in angular-cli.

**Comment by ljharb** [comment]
Fixed in 68a081d1c7ff6e0fb58aeff4b6ac06aada7812c4; will be released shortly as v1.3.1.
